lib/driver.js

```diff
@@ -468,6 +470,12 @@ class IosDriver extends BaseDriver {
     }
   }
 
+  async startIWDP () {
+    if (this.opts.startIWDP) {
+      await IWDP.start(this.opts.udid);
```

Comment:
Yeah that's a mistake. I think that method used to be static and now it's a class. Good catch.